ref_arch/real_time_dashboards.rst

```diff
@@ -30,8 +30,7 @@ all the details in one place. If you've followed our installation instructions f
 Citus on either a single or multiple machines you're ready to try it out.
 
 Data Model
-----------
-
```

Comment:
What is the change here? Visually I see no difference between this change and current docs?
https://docs.citusdata.com/en/v6.0/tech_soln/real_time_dashboards.html#data-model
https://citus-doc-preview-pr-253.herokuapp.com/ref_arch/real_time_dashboards.html#data-model